### PR TITLE
Add Homescreen page

### LINKED ISSUE #20: Homescreen
#### Screenshots

image

#### Tasks

Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.

- [x] Logo
- [x] Background
- [x] Buttons

### PR DESCRIPTION
**Pull request by Maurizio-F**
fixes #20 

heroku storybook preview:
https://movie-night-maurizio-f--56veod.herokuapp.com/storybook/?path=/story/page-homescreen--homescreen-page

### CODE REVIEW

File: src/app/pages/Homescreen/HomescreenElement.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+function HomescreenElement(): JSX.Element {
+  return (
+    <svg
+      data-name="Komponente 12 – 2"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (jujuzitroni) [inline src/app/pages/Homescreen/HomescreenElement.tsx]: can be removed

File: src/app/components/HomescreenLink/HomescreenLink.module.css [hunk 2]
```diff
@@ -1,10 +1,15 @@
 .homescreenLink {
   font-family: var(--font-family);
+  font-size: 2em;
+  background-color: var(--secondary-bg-color);
   color: var(--text-color);
   display: block;
   padding: 0.5em 0.9em;
   width: 9em;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (lajanzen) [inline src/app/components/HomescreenLink/HomescreenLink.module.css]:
```suggestion
  margin: 0 auto 1.2em
```

File: src/app/components/HomescreenLink/HomescreenLink.module.css [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
   text-align: center;
   border: 0.06em solid var(--tertiary-color);
   border-radius: 0.325em;
+  margin-bottom: 1.2em;
```
Comment (lajanzen) [inline src/app/components/HomescreenLink/HomescreenLink.module.css]:
```suggestion

```

File: src/app/pages/Homescreen/HomescreenElement.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+          stroke="none"
+        />
+        <path
+          d="M 209.6533508300781 2.346260070800781 L 391.2400512695312 79.08888244628906 L 414.660400390625 2.484382629394531 L 209.6533508300781 2.346260070800781 M 194.8247375488281 -0.6637344360351562 L 417.8762512207031 -0.5134506225585938 L 418.383544921875 0.5674591064453125 L 393.1363525390625 83.14720916748047 L 194.8247375488281 -0.6637344360351562 Z"
+          stroke="none"
+          fill="var(--tertiary-color)"
+        />
+      </g>
+      <g
+        data-name="Pfad 11"
```
Comment (jujuzitroni) [inline src/app/pages/Homescreen/HomescreenElement.tsx]: can be removed

File: src/app/pages/Homescreen/HomescreenElement.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function HomescreenElement(): JSX.Element {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 428 310.982">
+      <g data-name="Pfad 41" fill="#1d3557">
```
Comment (lajanzen) [inline src/app/pages/Homescreen/HomescreenElement.tsx]: Maybe you have some color variables for the fill color?

File: src/app/pages/Homescreen/HomescreenElement.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        <path
+          d="M 427.5 310.4823913574219 L 0.5 310.4823913574219 L 0.5 0.5576571226119995 L 427.5 47.24946594238281 L 427.5 310.4823913574219 Z"
+          stroke="none"
+        />
+        <path
+          d="M 1 1.115264892578125 L 1 309.9823913574219 L 427 309.9823913574219 L 427 47.69769287109375 L 1 1.115264892578125 M 0 0 L 428 46.80108642578125 L 428 310.9823913574219 L 0 310.9823913574219 L 0 0 Z"
+          stroke="none"
+          fill="#707070"
```
Comment (lajanzen) [inline src/app/pages/Homescreen/HomescreenElement.tsx]: Same here with the color :)

File: src/app/pages/Homescreen/Homescreen.module.css [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  align-content: center;
+}
+
+.footer {
+  position: fixed;
+  overflow: hidden;
+  width: 100%;
+  margin: 0 auto;
+  bottom: 0;
+  left: 0;
```
Comment (jujuzitroni) [inline src/app/pages/Homescreen/Homescreen.module.css]: can this be condensed into one line?

**Review by lajanzen — APPROVED** [review]
Just a few suggestions :)

**Review by jujuzitroni — COMMENTED** [review]
gj!

File: src/app/pages/Homescreen/Homescreen.tsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    <div className={styles.container}>
+      <main>
+        <img className={styles.logo} src="./logoWithText.png" alt="" />
+      </main>
+      <div className={styles.bottom}>
+        <HomescreenLink text={"Planen"} link={""} />
+        <HomescreenLink text={"Zufallsgenerator"} link={""} />
+        <HomescreenElement />
+      </div>
+    </div>
```
Comment (jujuzitroni) [inline src/app/pages/Homescreen/Homescreen.tsx:17]: why did you change the footer into a div? a footer seems right here and is more specific.

**Review by MaxMuecket — APPROVED** [review]
lgtm 👌

File: src/app/pages/Homescreen/Homescreen.tsx (hunk 8 above)
Comment (Maurizio-F) [reply src/app/pages/Homescreen/Homescreen.tsx:17]: I thought if I also work without a header, I prefer to stay strict and also do not use a footer :)
